core/src/network/rpc/client.rs

```diff
@@ -649,12 +649,9 @@ impl<D: Database> Client<D> {
 			.with_retries(|client| {
 				let cells = cells.clone();
 				async move {
-					client
-						.rpc
-						.kate
-						.query_proof(cells.to_vec(), Some(block_hash))
+					query_proof(&client.client, cells.to_vec(), Some(block_hash))
 						.await
-						.map_err(|error| subxt::Error::Other(format!("{error}")))
+						.map_err(|error| subxt::Error::Other(format!("{:?}", error)))
 						.map_err(Into::into)
```

Comment:
Since now query proof returns subxt error, we can replace code with, lets check other related changes also
```rust
Ok(query_proof(&client.client, cells.to_vec(), Some(block_hash)).await?) 
```

for format, we can also use 
```rust
format!("{error:?}")
```

if needed